roles/nvidia-docker/tasks/main.yml

```diff
@@ -0,0 +1,72 @@
+- name: "[nvidia-docker] Install GCCv10"
+  package:
+    name: gcc10.x86_64
+    state: present
+
+- name: "[nvidia-docker] Save previos GCC binary"
```

Comment:
I'm a bit worried about messing with GCC defaults. Can we provide an environment variable to the NVIDIA driver installer to tell it to use `/bin/gcc10-gcc` as it's compiler? Usually you can do something like
```
GCC=/bin/gcc10-gcc /root/NVIDIA-installer.run
```
to cause it to pick that as the preferred version of GCC, that way we don't have to change system defaults.

We definitely have a weird case here since the kernel is built with new GCC, but the rest of userland is built with old GCC, so we don't want to replace the default GCC - just use the new one for building kernel-related things

Reply:
Used alternatives for this one + always switch to gcc7 as the last task in this role.